README.md

```diff
@@ -1,2 +1,33 @@
 # hadoop-router-fs
-hadoop-router-fs provides an Hadoop [FileSystem](https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/fs/FileSystem.html) implementation that transforms URIs and routes file system operations between Hadoop file systems according to the path URI in the operation. 
+hadoop-router-fs provides an Hadoop [FileSystem](https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/fs/FileSystem.html) implementation that transforms URIs and routes file system operations between Hadoop file systems according to the path URI in the operation.
+
+RouterFS does a simple prefix replacement according to mapping configurations.
+
+## Mapping configuration structure
+
+Mapping configurations are Spark properties of the following form: 
+`routerfs.mapping.${toFsScheme}.${mappingIdx}.${replace/with}=${path-prefix}`
+
+Examples:
+```properties 
+routerfs.mapping.lakefs.1.replace=s3a://bucket/dir/ # mapping src
+routerfs.mapping.lakefs.1.with=lakefs://example-repo/dev/ # mapping dst
+routerfs.mapping.lakefs.2.replace=s3a://bucket/prefix # mapping src
+routerfs.mapping.lakefs.2.with=lakefs://example-repo/dev/prefix # mapping dst
+```
+Make sure your source prefix ends with a slash when needed. 
```

Comment:
This does not support regexps.  This is a big limitation!  Among other things:

* Only prefix matching is supported, no exact matches (see previous comment).  Any pattern and especially replacement without trailing punctuation risks behaving very strangely.
* Many very useful replacements are impossible.  For instance, `replace=s3a://my-bucket/user/<USER>/` with `with=lakefs://my-repo/user-<USER>/` needs a regexp to say
  ```conf
  routerfs.mapping.lakefs.3.replace=s3a://my-bucket/user/([^/]*)/
  routerfs.mapping.lakefs.3.with=lakefs://my-repo/user-\1/
  ```
  or similar stuff for mapping just the `data/` "subdirectory" of every user.

Since regexps are no harder to use (the previous examples _would work unchanged_ even if interpreted as regexp replacements.
* Doesn't require explicit "default" configuration support.

Why make this change?